e2e_playwright/conftest.py

```diff
@@ -612,9 +612,19 @@ def wait_for_app_run(page: Page, wait_delay: int = 100):
     """Wait for the given page to finish running."""
     # Add a little timeout to wait for eventual debounce timeouts used in some widgets.
     page.wait_for_timeout(155)
-
+    # Make sure that the websocket connection is established.
+    page.wait_for_selector(
+        "[data-testid='stApp'][data-test-connection-state='CONNECTED']",
```

Comment:
nit: for `data-teststate` we use camel-case states, e.g. `running` and `notRunning`. I guess it would make sense to use the same casing here instead all-uppercase

Reply:
Yeah, the inconsitency is already on the object level of ScriptState vs ConnectionState. It might not be the best Idea to touch this in this PR (since we also send out these values for host communication). 

https://github.com/streamlit/streamlit/blob/9b388606f1e6fa290b8512caf0d3b7c0eed954af/frontend/app/src/connection/ConnectionState.ts#L17

https://github.com/streamlit/streamlit/blob/9b388606f1e6fa290b8512caf0d3b7c0eed954af/frontend/lib/src/ScriptRunState.ts#L17

Maybe the best solution for the tests is to just lowercase everything?

Reply:
hmhmhm that's unfortunate. Would lowercasing in the tests work though? Is the playwright `selector` case insensitive? If not, let's leave it like it is. At least it then is in sync with the enums

Reply:
Yeah, but maybe its better to leave it as is and potentially look into unifying casing for enum values in a later PR